.github/workflows/ci_workable.yml

```diff
@@ -0,0 +1,275 @@
+        uses: actions/checkout@v3
+
+      - name: Fetch History
+        run: git fetch --prune --unshallow
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: "go.mod"
+
+      - name: Find the Go Cache
+        id: go
+        run: |
+          echo "::set-output name=build-cache::$(go env GOCACHE)"
+          echo "::set-output name=mod-cache::$(go env GOMODCACHE)"
+
+      - name: Cache the Go Build Cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go.outputs.build-cache }}
+          key: ${{ runner.os }}-build-${{ github.sha }}-${{ hashFiles('**/go.sum') }}
+
+      - name: Cache Go Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go.outputs.mod-cache }}
+          key: ${{ runner.os }}-mod-${{ github.sha }}-${{ hashFiles('**/go.sum') }}
+
+      - name: Add setup-envtest
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          setup-envtest use ${{env.KUBERNETES_VERSION}} -p env --os $(go env GOOS) --arch $(go env GOARCH)
+
+      - name: Cache envtest binaries
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/.local/share/kubebuilder-envtest/
+          key: ${{ runner.os }}-kubebuilder-${{env.KUBERNETES_VERSION}}
+
+      - name: Run Unit Tests
+        run: |
+          export KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT=true
+          source <(setup-envtest use ${{env.KUBERNETES_VERSION}} -p env --os $(go env GOOS) --arch $(go env GOARCH))
+          make test
+
+  set-ref:
+    if: github.repository == 'Workable/external-secrets'
+    name: retrieve the actual build branch
+    needs:
+      - unit-tests
+    runs-on: ubuntu-latest
+    outputs:
+      ref: ${{ steps.set-branch.outputs.ref }}
+    steps:
+      - name: set the build branch
+        id: set-branch
+        shell: bash
+        run: |
+          if [[ "${{ inputs.manual_build_branch }}" == "" ]]; then
+            ACTUAL_BRANCH=${{ github.ref_name }}
+          else
+            "ACTUAL_BRANCH=${{ inputs.manual_build_branch }}"
+          fi
+          echo "ref=${ACTUAL_BRANCH#workable_feature/*}" >> $GITHUB_OUTPUT
+  
+  set-tag:
+    if: github.repository == 'Workable/external-secrets'
+    name: set the image tag
+    needs:
+      - set-ref
+    uses: Workable/external-secrets/.github/workflows/common_image_tagging_workflow.yaml@main
+    with:
+      push_to_prd: ${{ inputs.push_to_prd == true }}
+      tag: ${{ inputs.tag }}
+      ref: ${{ needs.set-ref.outputs.ref }}
+
+  build-and-push-docker-image:
+    if: github.repository == 'Workable/external-secrets'
+    environment: Workable
+    name: Build and push docker image
+    needs:
+      - set-ref
+      - set-tag
+    uses: Workable/external-secrets/.github/workflows/common_build_and_push_to_gcr.yaml@main
+    with:
+      dockerfile: Dockerfile_workable
+      build_branch: ${{ needs.set-ref.outputs.ref }}
```

Comment:
I think that this will break with the change above. Should we do the change in line 220 instead?

Reply:
I think the ouput name remains [ref](https://github.com/Workable/external-secrets/pull/3/files#diff-6c3e81e0026a0c0f4a80876122e2d4ce8bd8275037317bc61d23035d335fa0bcR209)

```
 echo "ref=${ACTUAL_BRANCH#workable_feature/*}" >> $GITHUB_OUTPUT
```

Reply:
addressed as discussed offline